sa/sa.go

```diff
@@ -1267,13 +1341,24 @@ func (ssa *SQLStorageAuthority) getNewIssuancesByFQDNSet(fqdnSets []setHash, ear
 // exists in the database
 func (ssa *SQLStorageAuthority) FQDNSetExists(ctx context.Context, names []string) (bool, error) {
 	var count int64
+	queryStart := ssa.clk.Now()
+	nameHashes := hashNames(names)
```

Comment:
Ah, looks like you did do the reuse here. I think, as above, let's just put it inline (if we were going to keep it as a variable name I'd point out that `nameHashes` should be `namesHash`)

Reply:
Ok. Will revert this diff and the other one.

Reply:
Addressed in bce7482